letsencrypt/controllers/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from . import letsencrypt
```

Comment:
[This should be `main` instead.](https://www.odoo.com/documentation/8.0/reference/guidelines.html#file-naming)